src/accountsdb/db.zig

```diff
@@ -3465,8 +3477,13 @@ test "geyser stream on load" {
     const snapdir = tmp_dir_root.dir;
     const snapshot_files = try findAndUnpackTestSnapshots(2, snapdir);
 
-    var snapshots = try FullAndIncrementalManifest.fromFiles(allocator, logger, snapdir, snapshot_files);
-    errdefer snapshots.deinit(allocator);
+    const snapshots = try FullAndIncrementalManifest.fromFiles(
```

Comment:
should ideally stick to one name for all of these

Reply:
b25e2e31dba827f6809c790866d4c2c3cc288920